www-cronrobot/public/images/dashboard/icons/dashboard.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
it's difficult to know the link goes to a dashboard, it looks like an exit lol

![image](https://user-images.githubusercontent.com/8007722/132255740-17577038-cf77-4323-bd4e-9c721de66625.png)

Reply:
oh alright, will replace with a grafana logo